entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -9,99 +9,218 @@ uid: core/get-started/aspnetcore/new-db
 
 # Getting Started with EF Core on ASP.NET Core with a New database
 
-In this tutorial, you build an ASP.NET Core MVC application that performs basic data access using Entity Framework Core. You use migrations to create the database from your EF Core model.
+In this tutorial, you build an ASP.NET Core MVC application that performs basic data access using Entity Framework Core. The tutorial uses migrations to create the database from the data model.
+
+You can follow the tutorial by using Visual Studio 2017 on Windows, or by using the .NET Core CLI with any code editor on any platform that .NET Core supports:
+
+# [Visual Studio](#tab/visual-studio)
+
+Choose this tab for Visual Studio 2017 instructions.
```

Comment:
We've never had tabs just to introduce tabs. I think it makes the article more confusing but that's probably because I know what tabs mean. It made me think the link [View this article's sample on GitHub](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb) was necesary if I was using VS because it's under the tab.

If you keep this tab ( I recommend removing), don't dup the view this article link.

Reply:
The view this article link has to be tabbed because the projects are different -- VS uses SQL Server, CLI uses SQLite.  Perhaps the explanation above the tabs is enough, I'll remove the explanations inside the tabs.  This would be a good candidate for an AB test experiment.

Reply:
I can do it this way without tabs:

>You can follow the tutorial by using Visual Studio 2017 on Windows, or by using the .NET Core CLI on Windows, macOS, or Linux.

>View this article's sample on GitHub:
>* [Visual Studio 2017 with SQL Server](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb)
>* [.NET Core CLI with SQLite](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/AspNetCoreNewDb.Sqlite/EFGetStarted.AspNetCore.NewDb).